lom/Makefile

```diff
@@ -7,7 +7,7 @@ UID = $(shell id -u)
 GUID = $(shell id -g)
 
 
-GO := /usr/local/go/bin/go
+GO := /usr/local/go1.20.3/go/bin/go
```

Comment:
This is creating error on my build  container. I have to switch to /usr/local/go/bin/go

Reply:
Rebuild slave container